conjur/login/login_controller.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+"""
+LoginController module
+
+This module is the controller that facilitates all login actions
+required to successfully configure the netrc
+"""
+
+# Builtins
+import getpass
+import logging
+import sys
+
+from conjur.init import ConjurrcData
+from conjur.utils import Utils
+import conjur.constants
+
+class LoginController:
+    """
+    LoginController
+
+    This class represents the Presentation Layer for the LOGIN command
+    """
+    password = None
+    credential_data = None
+    login_logic = None
+    ssl_verify = None
+
+    def __init__(self, ssl_verify, password, credential_data, login_logic):
+        self.ssl_verify = ssl_verify
+        if self.ssl_verify is False:
+            Utils.insecure_message()
```

Comment:
Method should be named as an action describing what it does, not as a noun.
Replace with similar to:
```suggestion
            Utils.print_insecure_message()
```

Reply:
actually, it would be better to keep all the printing to the user inside the `controller` (remember?)
So you need something like:
```suggestion
            print(Utils.get_insecure_message())
```

Reply:
Also, isn't it relevant for all the commands? Why is it here?
It should be outside in one place, before starting to handle any command.

Reply:
removing this message because got feedback it is bad UX and annoying

Reply:
🤣